ECIP-1094: Integration of VeriBlock Proof-of-Proof into Ethereum Classic

max-sanchez opened the pull request:
This ECIP proposes the integration of VeriBlock PoP security to protect Ethereum Classic from long-range 51% reorganizations and 51% attack censorship attacks against smart contract protocols.

q9f commented on ECIP-VeriBlock-PoP-51%-Protection.md:
```diff
@@ -0,0 +1,569 @@
+---
+lang: en
+ecip: ECIP ????
```
```suggestion
ecip: ECIP 1094
```

q9f replied:
please rename the file to ecip-1094.md

q9f commented on ECIP-VeriBlock-PoP-51%-Protection.md:
```diff
@@ -0,0 +1,569 @@
+---
+lang: en
+ecip: ECIP ????
+title: Proof-of-Proof 51% Attack Prevention
```
```suggestion
title: VeriBlock Proof-of-Proof 51%-Attack Prevention
```

q9f commented on ECIP-VeriBlock-PoP-51%-Protection.md:
```diff
@@ -0,0 +1,569 @@
+ecip: ECIP ????
+title: Proof-of-Proof 51% Attack Prevention
+status: Draft
+type: Standards Track
```
please add `category: core`

q9f commented on ECIP-VeriBlock-PoP-51%-Protection.md:
Please add a license footer. See template for examples.

max-sanchez replied:
Added, thanks!

q9f commented:
Also, could you try to attend the next call? See #333

q9f commented:
Can you change the filename to `ecip-1094.md` and the assets also?

q9f commented on ECIP-VeriBlock-PoP-51%-Protection.md:
```diff
@@ -0,0 +1,575 @@
+---
+lang: en
+ecip: ECIP 1094
```
```suggestion
ecip: 1094
```

q9f commented on ECIP-VeriBlock-PoP-51%-Protection.md:
```diff
@@ -0,0 +1,575 @@
+category: core
+status: Draft
+type: Standards Track
+discussions-to: https://github.com/max-sanchez/ECIPs-1/issues/?
```
Uh, this requires some valid link. Easiest would be to create an issue here on the repository.

max-sanchez commented:
@q9f Moved and renamed the spec and it's correlating assets, and added a valid discussions link to a new issue. Let me know if there are any other formatting changes needed.
